html/changelogs/implant_eggs_larvae_scans.yml

```diff
@@ -0,0 +1,7 @@
+author: Vrow
+
+delete-after: True
+
+changes:
+  - bugfix: "Fixes medscans/body scanners/operating consoles thinking there's something else when someone only has an implant in them."
```

Comment:
```suggestion
  - bugfix: "Fixes the body scanner subsystem from thinking there is an unknown object something else when someone has an implant."
```

Reply:
This is just requesting changes for the sake of doing so, and being unnecessarily picky. The changelog is clear in what it changes, which is the point of it.